### PR TITLE
adding chore as a new template. Fixes #11263

### LINKED ISSUE #11263: Create a chore task issue template
There are several issues that don't fit on the existing templates. Currently, we have only bugs and features templates. We need at least a "chore" template.

image

Impacted by this bug? Give it a 👍.

### PR DESCRIPTION
**Pull request by varodrig**
**Description of your changes:**
Resolves https://github.com/kubeflow/pipelines/issues/11263
Adding a chore template

**Checklist:**
- [ X] You have [signed off your commits](https://www.kubeflow.org/docs/about/contributing/#sign-off-your-commits)
- [X ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/CHORE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Chore
+about: Create a Chore on Kubeflow Pipelines
+title: " <Chore Name>"
+labels: "type : chore, area/backend, area/frontend, area/sdk"
+---
+
+## Describe the chore
```
Comment (hbelmiro) [inline .github/ISSUE_TEMPLATE/CHORE.md:8]:
```suggestion
## Chore description
```

File: .github/ISSUE_TEMPLATE/CHORE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Describe the chore details and why it's needed.
```
Comment (hbelmiro) [inline .github/ISSUE_TEMPLATE/CHORE.md:10]:
```suggestion
<!-- Describe the chore details and why it's needed.  -->
```

File: .github/ISSUE_TEMPLATE/CHORE.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+## Describe the chore
+
+Describe the chore details and why it's needed. 
+
+
+## Component
+
+Describe components affected/updated
```
Comment (hbelmiro) [inline .github/ISSUE_TEMPLATE/CHORE.md:15]:
```suggestion
### Labels
<!-- Please include labels below by uncommenting them to help us better triage issues -->

<!-- /area frontend -->
<!-- /area backend -->
<!-- /area sdk -->
<!-- /area testing -->
<!-- /area samples -->
<!-- /area components -->

---

<!-- Don't delete the message below to encourage users to support your issue! -->
Love this idea? Give it a 👍. 
```

**Comment by varodrig** [comment]
@hbelmiro accepted suggestions and now ready for review.

**Comment by varodrig** [comment]
> Thank you @varodrig. I left some suggestions.

Thank you. Accepted suggestions and PR is now ready for review.

**Review by hbelmiro — COMMENTED** [review]
/lgtm

cc @HumairAK

**Comment by HumairAK** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**